app/design/frontend/Belodubrovskyi/Theme/Magento_Newsletter/templates/subscribe.phtml

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
+
+/** @var \Magento\Newsletter\Block\Subscribe $block */
+
+?>
+<div class="block newsletter">
+    <?= $block->getChildHtml( 'title_text');?>
```

Comment:
no need the space before the argument:
$block->getChildHtml( 'title_text');  
correct to 
$block->getChildHtml('title_text');

Reply:
Fixed